src/server.js

```diff
@@ -83,18 +83,30 @@ module.exports = function() {
 		}, app);
 	}
 
-	server.listen({
-		port: config.port,
-		host: config.host,
-	}, () => {
-		const protocol = config.https.enable ? "https" : "http";
-		var address = server.address();
-
-		log.info(
-			"Available at " +
-			colors.green(`${protocol}://${address.address}:${address.port}/`) +
-			` in ${colors.bold(config.public ? "public" : "private")} mode`
-		);
+	let listenParams;
+
+	if (typeof config.host === "string" && config.host.startsWith("unix:")) {
+		listenParams = config.host.replace(/^unix:/, "");
+	} else {
+		listenParams = {
+			port: config.port,
+			host: config.host,
+		};
+	}
+
+	server.listen(listenParams, () => {
+		if (typeof listenParams === "string") {
```

Comment:
It would be a bit clearer if both types were an object with a property `type`, so that you could do `if (listenParams.type === ...)`.
Alternatively, both objects with different properties and `if (listenParams.socket)`.

Reply:
Ideally yes, but `server.listen` can't accept an object like that.

Reply:
Ah, gotcha. Somehow I read this object as something different but I see now. All good.